libs/providers/go-feature-flag-web/src/lib/go-feature-flag-web-provider.ts

```diff
@@ -0,0 +1,269 @@
+import {
+  EvaluationContext,
+  FlagNotFoundError,
+  FlagValue,
+  Logger,
+  OpenFeature,
+  OpenFeatureEventEmitter,
+  Provider,
+  ProviderEvents,
+  ResolutionDetails,
+  StandardResolutionReasons,
+  TypeMismatchError,
+} from "@openfeature/web-sdk";
+import {
+  GoFeatureFlagAllFlagRequest,
+  GOFeatureFlagAllFlagsResponse,
+  GoFeatureFlagWebProviderOptions,
+  GOFeatureFlagWebsocketResponse,
+} from "./model";
+import axios from "axios";
```

Comment:
TOTALLY up to you, but I would recommend just using `fetch`. With a simple wrapper it will work pretty well. The only gotcha is I think by default it doesn't throw on non-200, but you can change that without much effort. It will keep you package size down at least, which is nice for client projects where every byte of JS counts.

Reply:
You could also totally do this as a new issue :shrug: